### DISCUSSION TITLE
spf and dkim settings cannot be verified and mail is not sent

### DISCUSSION THREAD

**Discussion by mucktobrak** [postalserver/postal#2416] [Help with using Postal]
I'm sure I set up the system properly, but it doesn't send mail. soft fail error.

**Comment by sinnbeck** [comment]
I suggest adding some more information if people are to help you out. What is the exact error ?

**Comment by willpower232** [comment]
You've described two problems.

DNS verification can be tricky depending on your setup, you may need to change this value in your config file

https://github.com/postalserver/postal/blob/5222263484f2a5889653859db6aeb7438f116357/config/postal.defaults.yml#L17

Softfails can be caused for many reasons, are you able to share a screenshot of the error message you're seeing?

**Comment by mucktobrak** [comment]
Can you recommend vps with smtp port open?

**Reply by uhlhosting** [reply]
@mucktobrak try Contabo , OVH, and depends on your configs, I am also selling VPSes.  And all our ports are allowed.
